frontend/src/lib/test/fixtures/cart_item.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Lets make this a `.ts` file and when you export it you can do `as CartItem` there instead of where you pass it in as props to the component